src/python/espressomd/profiler.pyx

```diff
@@ -0,0 +1,31 @@
+cimport c_profiler
+
+
+def begin_section(name):
+    """
+    Start named section in profiler.
+
+    Parameters
+    ----------
+
+    name: obj:`str`
```

Comment:
```name : obj:`str```

Reply:
Does this affect the result?

Reply:
idk

Reply:
Is this style documented somewhere? It does not produce a sphinx warning not to have the space

Reply:
https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Can this be enforces automatically? Otherwise we should not require it.

Reply:
I think no.